packages/flutter_tools/lib/src/doctor.dart

```diff
@@ -506,7 +530,7 @@ class DeviceValidator extends DoctorValidator {
       if (diagnostics.isNotEmpty) {
         messages = diagnostics.map((String message) => new ValidationMessage(message)).toList();
       } else {
-        messages = <ValidationMessage>[new ValidationMessage('None')];
+        messages = <ValidationMessage>[new ValidationMessage.hint('No devices found')];
```

Comment:
IIRC, in the case where we detected a device but it wasn't authorized, we end up saying that there's no devices, even though there is. You may wish to double check how we handle that case while you're here.

Reply:
rephrased

Reply:
rephrased